packages/ui/src/primitives/tailwind/Checkbox/index.tsx

```diff
@@ -48,8 +48,14 @@ const Checkbox = ({ className, containerClassName, label, value, onChange, disab
   )
 
   return (
-    <div className={twMerge('flex w-full items-center gap-4', containerClassName)}>
-      <input type="checkbox" onChange={(e) => onChange(e.target.value as any)} className={twClassName} {...rest} />
+    <div className={twMerge('relative flex w-full items-center gap-4', containerClassName)}>
+      <input
+        type="checkbox"
+        checked={value}
+        onChange={(e) => onChange(e.target.checked as any)}
+        className={twClassName}
+        {...rest}
+      ></input>
```

Comment:
let's get it reverted since it is handled by https://github.com/EtherealEngine/etherealengine/pull/10322

Reply:
lets keep it, these changes can be overwritten as needed